pkgs/tools/security/pcsclite/default.nix

```diff
@@ -5,10 +5,10 @@ stdenv.mkDerivation rec {
   name = "pcsclite-${version}";
   version = "1.8.23";
 
+  outputs = [ "bin" "out" "dev" "doc" "man" ];
+
   src = fetchurl {
-    # This URL changes in unpredictable ways, so it is not sensible
-    # to put a version variable in there.
-    url = "https://pcsclite.apdu.fr/files/pcsc-lite-1.8.23.tar.bz2";
+    url = "https://pcsclite.apdu.fr/files/pcsc-lite-${version}.tar.bz2";
```

Comment:
This was done in 4d5565e87e134ccdb9e245c1753b086e2b0a20e7. Could you rebase?

Reply:
Does it matter? It should still merge cleanly.

Reply:
Not much; it only makes history and blame view a bit confusing.

Reply:
I will rebase it before merging, do not have enough free space for another build.